bots/validatespeakerscornerissue.py

```diff
@@ -0,0 +1,118 @@
+import os
+from datetime import date, timedelta
+from github import Github
+import arxiv
+
+VSF_BOT_TOKEN = os.getenv("VSF_BOT_TOKEN")
+ISSUENUMBER = os.getenv("ISSUENUMBER")
+
+g = Github(VSF_BOT_TOKEN)
+repo = g.get_repo("virtualscienceforum/virtualscienceforum")
+issue = repo.get_issue(number=ISSUENUMBER)
+
+def CheckDate(dateStr, timeStr):
```

Comment:
The Python tradition for functions and variable names is `words_with_underscores` and not `camelCase`, let's stick to that.